MekHQ/src/mekhq/campaign/parts/PartInUse.java

```diff
@@ -1,17 +1,13 @@
 package mekhq.campaign.parts;
 
 import java.util.Objects;
-import java.util.regex.Pattern;
 
 import megamek.common.AmmoType;
 import mekhq.campaign.finances.Money;
 import mekhq.campaign.unit.Unit;
 import mekhq.campaign.work.IAcquisitionWork;
 
 public class PartInUse {
-    private static final Pattern cleanUp1 = Pattern.compile("\\d+\\shit\\(s\\),\\s"); //$NON-NLS-1$
```

Comment:
lol

Reply:
The purpose of this is to keep the parts overview from being cluttered by showing number of hits the part has taken.
![Screenshot from 2020-01-23 10-36-19](https://user-images.githubusercontent.com/16927464/73004145-6aa45a00-3dcc-11ea-9f0e-e4fa42f5d293.png)

Reply:
Woops, late night editing missed the Java regex difference. .NET regex that'd be an escaped `\` followed by `shit`. I'll find a better way than regex to remove those.

Reply:
I'm not going to pretend I've never snickered at it. My initial thought is a boolean parameter to getDetails() that would determine whether to include the hit data.

Reply:
I've added an overload to handle that situation. One interesting find is that PodSpace seems to do something a bit different; but ultimately it isn't affected by this change.